### ISSUE TITLE
Quicker feedback on count of accepted projects

### ISSUE THREAD

**Issue by zachsa** [SAEON/national-climate-change-systems#72]
Regarding https://github.com/SAEON/national-climate-change-response-database/issues/71, the home page was being refreshed as a means of ensuring that project counts increased correctly after changing a project status.

But the map on the home page may have inaccurate project counts because:

- The map draws on the same data as the charts use, which is truncated to the _**current year - 1**_ date
- The map omits projects that don't have location information. This may or may not be a factor depending on if a project can be accepted without location information (now or ever) but there are likely other reasons that the map will omit projects...

In short, the home page map is not intended as a mechanism for validating the project count for public users.

Currently the easiest way of validating project count for a public user (whilst logged in and making changes to projects) is to open the search page in a new incognito browser window. This is not very intuitive and could be improved on.
